common/models/access-token.js

```diff
@@ -101,7 +101,12 @@ module.exports = function(AccessToken) {
         if (err) {
           cb(err);
         } else if (token) {
-          token.validate(function(err, isValid) {
+          var possibleDate = new Date(token.created);
+          var isAValidDate = !isNaN(possibleDate.getTime());
+          if (isAValidDate) {
+            token.created = new Date(token.created);
+          }
```

Comment:
I have an application using connector-remote datasource for AccessToken, this was a temporal fix but I fixed the source in the connector module https://github.com/danilat/loopback-connector-remote/commit/db9446705dedabea6811a68faf4ce33e48a93beb